getGit.tex

```diff
@@ -185,361 +215,201 @@ \section{What, Why, and How?}
-        \vspace{0.5cm}
-	    \item{Git repositories hold special hidden files that describe your project's history}
-        \vspace{0.5cm}
-	    \item{You can perform `git` commands from within a repository}
-	\end{itemize}
-\end{frame}
-
+\section{Getting Started With Git}
 \begin{frame}
-	\frametitle{Commit: the basic unit of change}
-	\begin{itemize}
-	    \item{A commit is a thematically grouped set of changes}
-        \vspace{0.5cm}
-	    \item{Commits let us understand and manipulate our development history}
-        \vspace{0.5cm}
-	    \item{Group and name your commits well, so they're easy to work with.}
-	\end{itemize}
-\end{frame}
-
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	A is a commit, with a really bad name.
-
-\begin{verbatim}
-
-
-
-A  
-o
-
-
-
-\end{verbatim}
-\end{frame}
-
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	Your software project grows across time.
-\begin{verbatim}
-
-
-
-A  
-o
-
-
-
-\end{verbatim}
-\end{frame}
-
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	    Your software project grows across time.
-\begin{verbatim}
-
-
-
-A  B
-o--o
-   
-
-
-\end{verbatim}
-\end{frame}
-
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	    Your software project grows across time.
-\begin{verbatim}
-
-
-
-A  B  C
-o--o--o
-      
-     
-    
-\end{verbatim}
-\end{frame}
-
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	You develop many features on your project's branches.
-\begin{verbatim}
-
-
-
-A  B  C  
-o--o--o--\
-          \ 
-           \--o
-              D
-\end{verbatim}
-\end{frame}
-
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	You develop many features on your projects branches.
-\begin{verbatim}
-
-             
-              
-A  B  C     E
-o--o--o--\--o
-          \ 
-           \--o
-              D  
-\end{verbatim}
-\end{frame}
-
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	You develop many features on your projects branches.
-\begin{verbatim}
-                   F
-                /--o
-               /
-A  B  C     E /
-o--o--o--\--o/
-          \ 
-           \--o
-              D  
-\end{verbatim}
+    \frametitle{Git and Operating Systems}
+    \begin{itemize}
+        \item{Windows: Install gitbash.}
+        \item{Linux: Install through package manager.}
+        \item{Mac: Download git-osx-installer and run. }
+    \end{itemize}
+    \vspace{1cm} % generate some space between title and content
 \end{frame}
 
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	You develop many features on your projects branches.
-\begin{verbatim}
-                   F
-                /--o
-               /
-A  B  C     E /  G
-o--o--o--\--o/---o
-          \ 
-           \--o
-              D  
-\end{verbatim}
-\end{frame}
+% My super roughed up workflow is still there for now just so that slide has something
```

Comment:
This comment can be cut out, now that you've removed that "roughed up" workflow diagram.